tests/conftest.py

```diff
@@ -1,29 +1,56 @@
-from datetime import datetime
+from collections import namedtuple
 
 import pytest
-from slackclient import SlackClient as RealSlackClient
 from sqlalchemy import create_engine
 from sqlalchemy.orm import Session
 
-import karmabot.commands.topchannels
-from karmabot.db import db_session
+from karmabot.db.database import database
```

Comment:
Specially here importing `Database` you can give its instance another test connection string which might save you some mocking